docs/images/drawio/AKS_ScalarDB_Cluster_Direct_Kubernetes_Mode.drawio

```diff
@@ -0,0 +1,320 @@
+<mxfile host="app.diagrams.net" modified="2023-07-21T07:32:41.001Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" etag="8UMnNdPi2g1rvwuZ5w2Y" version="21.6.5" type="google">
```

Comment:
This is a diagram for AKS. We don't provide ScalarDB Cluster on the Azure Marketplace yet, but I added the diagram only for future releases.